CONTRIBUTING.md

```diff
@@ -1,52 +1,57 @@
-# 贡献指南
+# Contribution Guidelines
 
-欢迎您对本仓库做出贡献！非常感谢您的帮助，以下是一些建议和指导，以确保贡献的顺利进行。
+Welcome to contribute to this repository! Your help is greatly appreciated and here are some suggestions and guidance to ensure the contribution goes smoothly.
 
-## 贡献方式
+## Contribution mode
 
-您可以通过以下方式贡献：
+You can contribute by:
 
-1. **报告问题**：发现了文档中的错误或有改进的地方？请创建一个 Issue 来报告问题。
-2. **提出改进**：如果您有本项目的建议，请通过 Issue 提出您的想法。
+1. **Reporting problems**： Found an error or improvement in the documentation? Please create an Issue to report the problem.
+2. **Propose improvements**： If you have suggestions for this project, please send them through an Issue.
```

Comment:
`please send them through an Issue.` 
them? maybe `send us through an issue` be matched

Reply:
don't forget to resove this conversation